src/reducers/cart_reducer.js

```diff
@@ -33,6 +33,7 @@ const cart_reducer = (state, action) => {
         image: product.image,
         price: product.price,
         max: product.inventory,
+        product: product._id
```

Comment:
Add product = productId, to the the cart items, so that the backend can handle the order correctly

Reply:
@akosasante is this coming from 
![image](https://github.com/NatachaKey/reactEcommerce/assets/103755090/afbcfb51-6b47-431c-b570-45fa948bb839)
?